app/shared/design-system/components/button.tsx

```diff
@@ -3,6 +3,7 @@ import { styled } from "../stitches.config";
 export const Button = styled("button", {
   // Reset
   all: "unset",
+  cursor: "pointer",
```

Comment:
there is a bunch of resources suggesting cursor should be default on a button

Reply:
e.g. https://ux.stackexchange.com/a/20985

Reply:
removed